README.md

```diff
@@ -284,6 +284,16 @@ In order to load the standard configuration file from Leiningen, add the
   Paths can also be passed as command line arguments. If the path is
   `-`, then the input is STDIN, and the output STDOUT.
 
+* `:extra-configs` -
+  additional config files that will also be imported and merged into
+  the base configuration. Is not recursive, so `:extra-configs` in
```

Comment:
It probably should be recursive.

Reply:
Yea I could see that; I was worried about how this might be confusing or hard to debug. Some odd corner cases I was thinking about:

* You pull in a config from a lib A, which references lib B, but you never pull in lib B's config, they end up missing or causing errors?
* Could ordering get messy? thinking of multiple libraries using some other base library
* Could the relative path references get messed up, depending on how they're pulled in?
* Probably some others I'm missing? 

If it's not recursive though, I could see inconveniences there as well. Not sure.